themes/default/content/registry/packages/pulumiservice/_index.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Pulumi Service
+title: Pulumi Cloud (Pulumi Service provider)
```

Comment:
i dont think we should change this line until the package name changes

Reply:
that's why i kept both names in the title for discoverability purposes. when is the package being updated?

Reply:
@gthuang i dont know, youll have to work with @meagancojocar and @EvanBoyle on that

Reply:
I don't think there is much of a downside to this small change. We don't have a plan for doing the work to rename the provider yet.

Reply:
alright if yall dont think it will be confusing don't want to block.

Reply:
thanks both, i will merge this PR